src/kbmod/filters/clustering_filters.py

```diff
@@ -145,49 +126,30 @@ def _build_clustering_data(self, result_data):
         # the division by height and width will be no-ops.
         coords = []
         for t in self.times:
-            coords.append((x_arr + t * vx_arr) / self.width)
-            coords.append((y_arr + t * vy_arr) / self.height)
+            coords.append(x_arr + t * vx_arr)
+            coords.append(y_arr + t * vy_arr)
         return np.array(coords)
 
 
-class ClusterPosAngVelFilter(DBSCANFilter):
-    """Cluster the candidates using their scaled starting position, trajectory
-    angles, and trajectory velocity magnitude.
-    """
+class ClusterPosVelFilter(DBSCANFilter):
+    """Cluster the candidates using their starting position and velocities."""
 
-    def __init__(self, eps, height, width, vel_lims, ang_lims, **kwargs):
+    def __init__(self, cluster_eps, cluster_v_scale=1.0, **kwargs):
         """Create a DBSCANFilter.
 
         Parameters
         ----------
-        eps : `float`
-            The clustering threshold.
-        height : `int`
-            The size of the image height (in pixels) for scaling.
-        width : `int`
-            The size of the image width (in pixels) for scaling.
-        vel_lims : `list`
-            The velocity limits of the search such that v_lim[1] - v_lim[0]
-            is the range of velocities searched. Used for scaling.
-        ang_lims : `list`
-            The angle limits of the search such that ang_lim[1] - ang_lim[0]
-            is the range of velocities searched.
+        cluster_eps : `float`
+            The clustering threshold in pixels.
+        cluster_v_scale : `float`
+            The relative scaling of velocity differences compared to position
+            differences. Default: 1.0 (no difference).
         """
-        super().__init__(eps, **kwargs)
-        if height <= 0.0 or width <= 0:
-            raise ValueError(f"Invalid scaling parameters y={height} by x={width}")
-        if len(vel_lims) < 2:
-            raise ValueError(f"Invalid velocity magnitude scaling parameters {vel_lims}")
-        if len(ang_lims) < 2:
-            raise ValueError(f"Invalid velocity angle scaling parameters {ang_lims}")
-        self.height = height
-        self.width = width
-
-        self.v_scale = (vel_lims[1] - vel_lims[0]) if vel_lims[1] != vel_lims[0] else 1.0
-        self.a_scale = (ang_lims[1] - ang_lims[0]) if ang_lims[1] != ang_lims[0] else 1.0
-        self.min_v = vel_lims[0]
-        self.min_a = ang_lims[0]
-
+        super().__init__(cluster_eps, **kwargs)
+        if cluster_v_scale <= 0.0:
+            # Avoid divide by zero.
+            cluster_v_scale = 1e-12
```

Comment:
Is this supposed to be == 0? But also if the scaling is 0, then it's equivalent to setting the velocities to 0 and defaulting to position-only matching, and that's shown in line 172 and 173 where they get multiplied together (velocities and v_scaling) so I'm a bit confused where does the division zero come to play?

Reply:
That's an error. I was originally scaling by 1/cluster_v_scale (to stay consistent with the old scales), but figured a multiplicative scaling makes more sense. I forgot to remove that check.

I've fixed it now.